tools/data/add_transcripts_from_gff3_to_annotations.pl

```diff
@@ -142,6 +144,7 @@ sub print_usage {
     [--success_log|-l <success log file>]
     [--error_log|-L <error log file>]
     [--skip|-s <skip id file>]
+    [--test|-x]
```

Comment:
The reason it is called `--test` is because the script never communicates with the Apollo instance in test mode. It simply parses the GFF3, converts to JSON and outputs it to STDOUT.